examples/chip-tool/commands/common/Commands.cpp

```diff
@@ -230,6 +232,27 @@ CHIP_ERROR Commands::RunCommand(int argc, char ** argv, bool interactive,
         return CHIP_ERROR_INVALID_ARGUMENT;
     }
 
+    if (strncmp(argv[1], "dumpstate", 9) == 0)
```

Comment:
Also, "dumpstate" is very generic (what state is being dumped?).  A more scoped-down name would be good.

Reply:
I'm thinking about whether we should expose this state to other classes.

I moved the logic to `icd queue` command now, do you have any ideas for this?